test/evp_pkey_provided_test.c

```diff
@@ -272,6 +272,20 @@ static int test_print_key_using_encoder(const char *alg, const EVP_PKEY *pk)
     return ret;
 }
 
+#ifndef OPENSSL_NO_EC
+static int test_print_key_using_encoder_public(const char *alg,
+                                               const EVP_PKEY *pk)
+{
+    int i;
+    int ret = 1;
+
+    for (i = PUB_TEXT; i < 6; i++)
```

Comment:
Would it be better to say `<= PUB_DER` here rather than `6`?

Reply:
Updated.. Updated the function above that also does this as well as put a comment above the defines saying not to modify them.